Podfile

```diff
@@ -135,7 +135,7 @@ end
 pre_install do |installer|
   installer.analysis_result.specifications.each do |s|
     if s.name == 'CocoaLumberjack'
-      s.swift_version = '4.2'
+      s.swift_version = '5.0'
```

Comment:
I don't know if I did this right.

Reply:
To be honest, I don't know what this does.